src/main/java/com/jcabi/github/StatusState.java

```diff
@@ -0,0 +1,5 @@
+package com.jcabi.github;
+
+public enum StatusState {
```

Comment:
@zygm0nt it's better to just put String fields inside Status interface and get rid of the enum